src/server/handlers/openapi/mojaloop/consentRequests/{ID}.ts

```diff
@@ -20,14 +20,25 @@
 
  * Google
  - Steven Wijaya <stevenwjy@google.com>
+ - Abhimanyu Kapur <abhi.kapur09@gmail.com>
  --------------
  ******/
 
 import { Request, ResponseToolkit } from '@hapi/hapi'
 import { Handler, Context } from 'openapi-backend'
-import { logger } from '~/shared/logger'
+import { consentRepository } from '~/repositories/consent'
+import { ConsentStatus } from '~/models/consent'
 
-export const put: Handler = async (context: Context, request: Request, h: ResponseToolkit) => {
-  logger.logRequest(context, request, h)
+export const put: Handler = async (
+  context: Context,
+  _request: Request,
+  h: ResponseToolkit
+) => {
+  const { authChannels, authUri } = context.request.body
+  consentRepository.updateConsentById(context.request.params.ID as string, {
```

Comment:
let add a comment here we are not waiting for a promise to resolve so this part of the code is executed async